compose/foundation/foundation/src/skikoMain/kotlin/androidx/compose/foundation/text/CupertinoTextFieldPointerModifier.skiko.kt

```diff
@@ -98,6 +98,12 @@ private fun getTapHandlerModifier(
         detectRepeatingTapGestures(
             onTap = { touchPointOffset ->
                 if (currentState.hasFocus) {
+                    // To show keyboard if it was hidden. Even in selection mode (like native)
+                    tapTextFieldToFocus(
```

Comment:
@dima-avdeev-jb 
It seems like I messed with the naming of that method in Common code in my previous request dedicated to that feature. Should I make another request for fixing that, or I can fix it there?

Reply:
You can fix it in this PR too